src/components/ProtectButtonSelector/index.tsx

```diff
@@ -1,17 +1,25 @@
-import React, { useEffect, useState } from "react"
+import React, { useEffect, useMemo, useState } from "react"
 import SimpleDropdown from '../SimpleDropdown'
 import FlashbotsProtectButton from 'protect-button'
 import Checkbox from '../Checkbox'
 import AlignItems from '../AlignItems/AlignItems'
 import GridBlock from '../GridBlock/GridBlock'
+import axios from "axios"
 
 type Builder = {
     name: string,
     rpc: string,
+    "supported-apis": Array<string>, // TODO: can we please change this to camelCase
+}
+
+const fetchDowgBuilders = async (): Promise<Array<Builder>> => {
```

Comment:
Maybe “fetchRegisteredBuilders”?

The phrase “DOWG builders” implies that these builders are part of

Reply:
good catch, done